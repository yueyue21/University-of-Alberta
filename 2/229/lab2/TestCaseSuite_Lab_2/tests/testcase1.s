.text
main:
bne $s0, $t1 main
