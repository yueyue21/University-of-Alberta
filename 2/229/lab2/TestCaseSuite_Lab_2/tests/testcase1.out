bne $16, $9, 0x10010000
